syntax = "proto3";

package NGenerativeBoltalka.Proto;

message TGenerativeResult {
    string Response = 1;
    float Score = 2;
    uint32 NumTokens = 3;
    optional string ExternalInfo = 4;
}

message TGenerativeResponse {
    repeated TGenerativeResult Responses = 1;
    bool BadWordInRequest = 2;
}
